guide/07-working-with-bigdata/02-making-your-data-accessible-to-ga-server.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>Learn more about output templates in the <a href="https://enterprise.arcgis.com/en/geoanalytics/latest/perform-analysis/output-templates-in-a-big-data-file-share.htm" target="_blank">Output templates in a big data file share</a> topic.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>